packages/react-component-library/src/components/AutocompleteE/AutocompleteE.tsx

```diff
@@ -11,8 +11,7 @@ import {
 } from '../SelectBase'
 import { NoResults } from './NoResults'
 import { useHighlightedIndex } from './hooks/useHighlightedIndex'
-import { useInput } from './hooks/useInput'
-import { useItems } from './hooks/useItems'
+import { useAutocomplete } from './hooks/useAutocomplete'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface AutocompleteEProps extends SelectBaseProps {}
```

Comment:
If we are going to keep the empty interface maybe we add an exclusion for this rule? 

I think i've also done this elsewhere.

Reply:
As discussed we will leave this as it is for now.